lib/src/modules/conditional.dart

```diff
@@ -524,10 +520,13 @@ class ConditionalAssign extends Conditional {
   @override
   List<Conditional> getConditionals() => [];
 
+  /// Assign [receiver] the value of the [driver].
+  ///
+  /// The [drivenSignals] is a collection of [receiver] in a Set.
```

Comment:
Since this is overridden via `@override`, it will inherit the doc string from the super class, so you dont need to add here.